subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -352,6 +353,26 @@ By default, repositories include everything and exclude nothing:
 It is possible to filter either by explicit _group_, _module_ or _version_, either strictly or using regular expressions.
 See link:{javadocPath}/org/gradle/api/artifacts/repositories/RepositoryContentDescriptor.html[RepositoryContentDescriptor] for details.
 
+=== Declaring content exclusively found in one repository
+
+Filters declared using the <<#sec:declaring-repository-filter,repository-level content filter>> are not exclusive.
```

Comment:
I would move this paragraph up, before the title. It concludes the section on content filters specific to a given repository.

Then, below the title, you have the rest that explains the purpose of exclusive filtering declaration.

Otherwise it feels weird to start a feature description by what another feature does not do.